main/internal/implementations/dragon_drop/http_dragondrop_oss_methods.go

```diff
@@ -7,52 +7,66 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"regexp"
 
 	log "github.com/sirupsen/logrus"
 )
 
+var githubRepoExp = regexp.MustCompile(`https://github.com/(.*).git`)
```

Comment:
This value is found only used within `getOrgAndRepo` and so it should probably be defined within that function itself.

Reply:
I thought that keeping the variable outside of the function was better because compile a regex has more cost than a regular variable definition.
But considering the job executes just one time and then destroy the instance, you got a point.